### PR TITLE
Add \^perp and \^bot as input for `ᗮ`, U+15EE

### PR DESCRIPTION
**Pull request by hrmacbeth**
In light of 
https://github.com/leanprover-community/mathlib/pull/5536
introducing the notation `ᗮ` for the orthogonal complement of a submodule in an inner product space, I propose introducing `\^perp` and `\^bot` as vscode input for `ᗮ`.

[Zulip](https://leanprover.zulipchat.com/#narrow/stream/116395-maths/topic/New.20linear.20algebra.20notation)

(note that an earlier version of the PR proposed `\perp` rather than `\^perp`; this has been updated after discussion.)

### CODE REVIEW

**Review by jcommelin — APPROVED** [review]
LGTM

File: translations.json [hunk 1]
```diff
@@ -523,7 +523,7 @@
     "parallel": "∥",
     "pa": "▰",
     "pm": "±",
-    "perp": "⊥",
+    "perp": "ᗮ",
```
Comment (hrmacbeth) [inline translations.json:526]:
```suggestion
    "perp": "⊥",
    "\perp": "ᗮ",
```

File: translations.json [hunk 2]
```diff
@@ -524,6 +524,7 @@
     "pa": "▰",
     "pm": "±",
     "perp": "⊥",
+    "\perp": "ᗮ",
```
Comment (hrmacbeth) [inline translations.json:527]:
```suggestion
    "^perp": "ᗮ",
```
oops

File: translations.json [hunk 3]
```diff
@@ -1022,6 +1023,7 @@
     "blacktriangleright": "▸",
     "blacktriangle": "▴",
     "bot": "⊥",
+    "^bot": "ᗮ"
```
Comment (hrmacbeth) [inline translations.json:1026]:
```suggestion
    "^bot": "ᗮ",
```

**Comment by eric-wieser** [comment]
Can you include the name of the unicode character in the PR title?

**Pull request merged**